RN-CNV-5438 virt security

lmandavi opened the pull request:
This PR is associated with https://issues.redhat.com/browse/CNV-5438.

lmandavi commented:
@stu-gott @fabiand - Please review this initial draft-write up and email me with any edits/changes/updates. I will fix the xref link resolution failure, but the content is available for you to review.

jean-edouard commented:
/lgtm

lmandavi commented:
SME review was completed. Requesting peer review.

ousleyp commented on virt/virt-2-4-release-notes.adoc:
```diff
@@ -18,6 +18,10 @@ include::modules/virt-what-you-can-do-with-virt.adoc[leveloffset=+2]
 
+
+//CNV-5438 - svirt-support-for-cnv
+Security has been significantly enhanced in this release. {VirtProductName} now supports SELinux with Mandatory Access Control (MAC) for isolating virtual machines (VMs). Previously, all VMs were managed by using privileged xref:../security/container_security/security-platform.adoc#security-deployment-sccs_security-platform[Security Context Constraints (SCC)]. Now, you can use less privileged cus ...
```
For the first sentence, I would say something like this (active voice): "This release features significant security enhancements."

There should be a bullet point at the beginning of the note (I think I missed this in your other PR as well).

One other nit: inconsistent pluralization when using the SCC acronym. It looks like all three instances should be `SCCs`.

lmandavi commented:
HI @ousleyp During the merge conflict resolution, I removed CNV-5439 content to reject that content and retain the CNV-5438 content. Is it okay to add back the CNV-5439 content while working with CNV-5438 branch? If that is okay, I can just add back the CNV-5439 content as part of the CNV-5439 content. Wanted to check with you what protocol we follow when the content for two jira stories collide during merge conflict. Thanks.

For this PR, peer review update has been completed and commits were squashed.

ousleyp commented:
> HI @ousleyp During the merge conflict resolution, I removed CNV-5439 content to reject that content and retain the CNV-5438 content. Is it okay to add back the CNV-5439 content while working with CNV-5438 branch? If that is okay, I can just add back the CNV-5439 content as part of the CNV-5439 content. Wanted to check with you what protocol we follow when the content for two jira stories collide during merge conflict. Thanks.
> 
> For this PR, peer review update has been completed and commits were squashed.

Hey @lmandavi -

The merge conflict happened when I merged your other PR into upstream/enterprise-4.5. To resolve the merge conflict, you should ensure that the release notes file contains *both* what I merged previously (the TLS content) and the security content from this PR. Removing the other PR's content will effectively undo our other merge. (The point of rebasing is to get all the updated content into your local branch.)

Let me know if you have any questions!

lmandavi commented:
Hi @Pan Ousley <pousley@redhat.com> - When the release notes adoc file is
opened in Atom during a merge conflict, Atom gave me a binary choice -
reject the text in red, and keep the text in green.
I learned that by using Atom, one is forced to make a binary choice (at
least I was). Therefore, I kept the text in green and clicked the button in
Atom to delete the text in red (which was the text for the other PR). The
lesson learned here - it is better to edit the merge conflict related text
in emacs editor so that it does not mandate what you can keep and what you
can reject. Using Atom to resolve the merge conflict text does not give me
flexibility to keep all of the text from the previous PR and the current PR
(that's my insight).

I understand what you're saying - it doesn't make sense to loose the text
from the previous PR because it was obviously merged and closed. In this
particular scenario, because that text was deleted, as a fix for this
particular scenario in order to get that lost text back into the rel notes
adoc file 1) I can create a new PR specifically to add that lost text back
into th file 2) Your suggestion on what to do. Thanks.


On Mon, Jul 13, 2020 at 8:53 AM Pan Ousley <notifications@github.com> wrote:

> HI @ousleyp <https://github.com/ousleyp> During the merge conflict
> resolution, I removed CNV-5439 content to reject that content and retain
> the CNV-5438 content. Is it okay to add back the CNV-5439 content while
> working with CNV-5438 branch? If that is okay, I can just add back the
> CNV-5439 content as part of the CNV-5439 content. Wanted to check with you
> what protocol we follow when the content for two jira stories collide
> during merge conflict. Thanks.
>
> For this PR, peer review update has been completed and commits were
> squashed.
>
> Hey @lmandavi <https://github.com/lmandavi> -
>
> The merge conflict happened when I merged your other PR into
> upstream/enterprise-4.5. To resolve the merge conflict, you should ensure
> that the release notes file contains *both* what I merged previously (the
> TLS content) and the security content from this PR. Removing the other PR's
> content will effectively undo our other merge. (The point of rebasing is to
> get all the updated content into your local branch.)
>
> Let me know if you have any questions!
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/openshift-docs/pull/23528#issuecomment-657542306>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ANHOAGIS22N2B75T4TUG22LR3L7SPANCNFSM4OTUEURA>
> .
>

lmandavi commented:
@pan - I added back the text for CNV-5439 that was lost during deletion in Atom when merge conflict occurred on Friday. Squashed the commits. Please review.
